diff --git a/polymetis/polymetis/python/polysim/envs/__init__.py b/polymetis/polymetis/python/polysim/envs/__init__.py
--- a/polymetis/polymetis/python/polysim/envs/__init__.py
+++ b/polymetis/polymetis/python/polysim/envs/__init__.py
@@ -1,4 +1,2 @@
 from .abstract_env import AbstractControlledEnv
 from .bullet_manipulator import BulletManipulatorEnv
-from .habitat_manipulator import HabitatManipulatorEnv
-from .mujoco_manipulator import MujocoManipulatorEnv
